manifests/shared/spec/support/fixture_helpers.rb

```diff
@@ -0,0 +1,35 @@
+module FixtureHelpers
+  def terraform_fixture_value(key, fixture)
+    YAML.load_file(fixtures_dir.join("#{fixture}-terraform-outputs.yml")).fetch("terraform_outputs_#{key}")
+  end
+
+  def copy_terraform_fixtures(target_dir, fixtures = %w(vpc bosh cf))
```

Comment:
maybe we should remove the default value for the fixtures from here as we define it everywhere